crates/bitwarden/src/auth/password/strength.rs

```diff
@@ -0,0 +1,99 @@
+use std::collections::HashSet;
+
+use zxcvbn::zxcvbn;
+
+const GLOBAL_INPUTS: [&str; 3] = ["bitwarden", "BW", "Bitwarden"];
```

Comment:
We use different inputs on the clients, is this intentional? https://github.com/bitwarden/clients/blob/0fba5810d9476db97b11f2c72213e9acda72f1f7/libs/common/src/tools/password-strength/password-strength.service.ts#L21-L24

Reply:
No.